dashboard/serializers.py

```diff
@@ -0,0 +1,52 @@
+"""
+Provides functionality for serializing a ProgramEnrollment for the ES index
+"""
+from decimal import Decimal
+
+
+class UserProgramSerializer:
+    """
+    Provides functions for serializing a ProgramEnrollment for the ES index
+    """
+    @staticmethod
+    def calculate_certificate_grade_average(certificates):
+        """
+        Calculates an average grade (integer) from a list of certificates with <1 decimal grades
+        """
+        return None if len(certificates) == 0 else round(
```

Comment:
i don't think we can do this yet b/c of the elasticsearch histogram limitation (can't use decimal values/ranges)